ckanext/datagovuk/templates/user/dashboard.html

```diff
@@ -6,8 +6,8 @@
       {% link_for _('Edit settings'), named_route='user.edit', id=user.name, class_='btn btn-default', icon='cog' %}
     </div>
     <ul class="nav nav-tabs">
-      {{ h.build_nav_icon('dashboard.datasets', _('My Datasets')) }}
-      {{ h.build_nav_icon('dashboard.organizations', _('My Organizations')) }}
+      {{ h.build_nav_icon('dashboard.datasets', h.humanize_entity_type('package', dataset_type, 'my label') or _('My Datasets'), icon='sitemap') }}
+      {{ h.build_nav_icon('dashboard.organizations', h.humanize_entity_type('organization', org_type, 'my label') or _('My Organizations'), icon='building-o') }}
```

Comment:
Do we mean to replace the placeholder "my_label"?

Reply:
set to more appropriate label